development/packaging.md

```diff
@@ -75,3 +76,8 @@ The `package.json` must contain the following keys
     },
     ```
  
+### Package Version Numbering
+
+Package numbers should follow the Semantic Versioning Scheme as laid out on [semver.org](https://semver.org/).
+
+Each component will maintain it's own numbering regardless of the core FlowForge project and only update it's version as needed.
```

Comment:
For the 'internal' components of the platform, I think we should say they need to keep their SemVer `major.minor` number in sync, but can have individual `fix` level releases.

Reply:
Updated to cover this